SH4RE/SH4RE/Search/SearchView.swift

```diff
@@ -133,6 +162,13 @@ struct ContentView_Previews: PreviewProvider {
 
 struct SearchView_Previews: PreviewProvider {
     static var previews: some View {
-        SearchView(tabSelection: .constant(1))
+        SearchView(tabSelection: .constant(1), searchModel: SearchModel())
+            .environmentObject(CurrentUser())
     }
 }
+//struct SearchView_Previews_Helper: View {
+//    var body: some View {
+//        SearchView(tabSelection: .constant(1), searchModel: SearchModel())
+//            .environmentObject(CurrentUser())
+//    }
+//}
```

Comment:
any reason for this comment?

Reply:
Mostly just because well need it once search is implemented I'm sure. I for rid of it because I got rid of a lot of the @Binding vars, but search still has a long way to go